woocommerce/woocommerce
# Product Details: not use `useBlockProps`  twice

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





While I was working on E2E tests, I noticed that for the Product Details block, the `useBlockProps` function is used twice:
- https://github.com/woocommerce/woocommerce/blob/121652ae0e44c06318eeb088f127f9be7a933f30/plugins/woocommerce-blocks/assets/js/atomic/blocks/product-elements/product-details/block.tsx/#L82-L83
- https://github.com/woocommerce/woocommerce/blob/121652ae0e44c06318eeb088f127f9be7a933f30/plugins/woocommerce-blocks/assets/js/atomic/blocks/product-elements/product-details/edit.tsx/#L23-L24

This causes an unnecessary duplicate structure:

![SAVWms.png](https://github.com/woocommerce/woocommerce/assets/4463174/9a89b887-1765-4e34-9c32-3a78485bc309)


This PR removes this duplicate structure.




### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go on `/wp-admin/site-editor.php?postType=wp_template&postId=woocommerce%2Fwoocommerce%2F%2Fsingle-product`.
2. Ensure that the Product Details block is added.
3. Open the Chrome Dev tools.
4. Click on `Elements`
5. Search `[data-type="woocommerce/product-details"]`
6. Ensure that you get only one result:

![image](https://github.com/woocommerce/woocommerce/assets/4463174/4628e88b-e6ba-4973-82cb-ecfa3f143fab)




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

Product Details: not use `useBlockProps`  twice

</details>
